x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/utils.test.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getDateString } from './utils';
+
+describe('getDateString', () => {
+  const mockDate = jest
+    .spyOn(global.Date, 'now')
+    .mockImplementation(() => new Date('1970-01-02').valueOf());
```

Comment:
I learned this from James when he used this in a Jest test recently!! 🤯 

As someone who loves setting 1. static dates and 2. Jan 1970, the beginning of the universe, I could NOT pass up using it here